addons/website_blog/views/website_blog_components.xml

```diff
@@ -187,19 +210,9 @@ Options:
 <!-- (Option) Sidebar Blog: Follow Us -->
 <template id="opt_sidebar_blog_index_follow_us" name="Follow Us" priority="1" inherit_id="website_blog.sidebar_blog_index" active="True">
     <xpath expr="//div[@id='o_wblog_sidebar']" position="inside">
-        <div class="o_wblog_sidebar_block pb-5">
+        <div class="o_wblog_sidebar_block pb-5" t-ignore="true">
             <h6 class="text-uppercase pb-2 mb-4 border-bottom fw-bold">Follow Us</h6>
-            <div class="o_wblog_social_links d-flex flex-wrap mx-n1 o_not_editable">
```

Comment:
I think we need to adapt that css styling that was set for all these icons for website_blog, That needs to be discussed with SHRA/LEBL. as the `s_social_media` has its own style.

Reply:
@ksbh-odoo in support of this [response](https://www.odoo.com/mail/message/621732492) we dont need to work on adapting styles, so simply remove the styles related to `o_wblog_social_links` as well.